homeassistant/components/tellduslive/entry.py

```diff
@@ -111,3 +112,16 @@ def _last_updated(self):
     def unique_id(self) -> str:
         """Return a unique ID."""
         return self._id
+
+    @property
+    def device_info(self):
```

Comment:
Device registry support can come in a separate PR.

Reply:
but device registry is only supported for entities with config_flow..

Reply:
Yes, so device registry support PR should follow this PR.